docs/reference/advanced_configuration.rst

```diff
@@ -177,5 +177,17 @@ Full configuration options:
                 generator:  sonata.media.generator.default
                 thumbnail:  sonata.media.thumbnail.format
 
+        # The buzz implementation is deprecated, use a PSR http-client instead
         buzz:
             connector:  sonata.media.buzz.connector.file_get_contents # sonata.media.buzz.connector.curl
+
+        http:
+            client:          'symfony_http_client'       # You need symfony/http-client for this
```

Comment:
```suggestion
            client: 'symfony_http_client'       # You need symfony/http-client for this
```

Reply:
Looking at the other configs, we use indention in most cases

Reply:
I saw in most of the recent PR that contributors are removing indentations.

Maybe the best things to do would be to decide about a standard.

Reply:
I vote for the unaligned version, since it reduces unrelated merge conflicts.